docs/web.md

```diff
@@ -0,0 +1,170 @@
+# Web Frameworks
+
+Since HTTP logging is a primary use case, Pino has first class support for the Node.js 
+web framework ecosystem.
+
++ [Pino with Fastify](#fastify)
++ [Pino with Express](#express)
++ [Pino with Hapi](#hapi)
++ [Pino with Restify](#restify)
++ [Pino with Koa](#koa)
++ [Pino with Node core `http`](#http)
+
+<a id="fastify"></a>
+## Pino with Fastify
+
+The Fastify web framework comes bundled with Pino by default, simply set Fastify's
+`logger` option to `true` and use `reply.log` for log messages that correspond 
+to each individual request:
+
+```js
+const fastify = require('fastify')({
+  logger: true
```

Comment:
you can add a comment about passing the logging options to fastify `logger: { opts }`.

Note that fastify 1 will still be using Pino v4, and we will move to Pino v5 in fastify 2.

Reply:
ok so I'll add two additional notes underneath the code I think

Reply:
this has been address https://github.com/pinojs/pino/pull/447/files#diff-0f426376b964e609c5c4618eb8bed3b2R227 and https://github.com/pinojs/pino/pull/447/files#diff-0f426376b964e609c5c4618eb8bed3b2R245

Reply:
this is addressed https://github.com/pinojs/pino/pull/447/files#diff-f7e8c7a96957c8be15b0c902dda2d07aR30 and https://github.com/pinojs/pino/pull/447/files#diff-f7e8c7a96957c8be15b0c902dda2d07aR33